### DISCUSSION TITLE
How to reset an openDTU ???

### DISCUSSION THREAD

**Discussion by ifficgo** [tbnobody/OpenDTU#2404] [Q&A]
Good afternoon,

I bought an OpenDTU 1 year ago...at the time I configured my HMS1600 but I was unable to connect the OpenDTU to my wifi network...I left the subject in suspended, since on my phone when I switched to the OpenDTU wifi I had the production information...

Today I am trying again to configure this OpenDTU correctly...but problem...to access the management parameters of the OpenDTU I am asked for network name and password ....and naturally there is no way to find them since 'at the time I was fiddling around trying to connect the OpenDTU to my WiFi network... I of course tried quite a few ones...

So 1/ is there a procedure to "reset" this access....I doubt it because I searched the net and couldn't find it...
2/ is it possible to reset it and start the installation from scratch...? 

This is an OpenDTU to follow an HMS1600

Thank you for your advices.

**Comment by ms49434** [comment]
You should consider to start from scratch due to the age of the installation. Please follow the instructions outlined in 
https://www.opendtu.solar/firmware/
https://www.opendtu.solar/firmware/flash_esp/ and
https://www.opendtu.solar/firmware/webinstall/

**Reply by ifficgo** [reply]
Thanks ms49434 for your answer...I'm not at all an IT expert but I'll try...Having an HMS1600, my openDTU is an ESP32 ? ESP32S3 ? or an ESP32S3 with integrated USB connection ?
I'll try to flash it through web flasher...

Thanks for your answer...
